lib/vmfloaty.rb

```diff
@@ -35,6 +35,7 @@ def run
       c.option '--token STRING', String, 'Token for pooler service'
       c.option '--notoken', 'Makes a request without a token'
       c.option '--force', 'Forces vmfloaty to get requested vms'
+      c.option '--json', 'Prints retrieved nodes in JSON format'
```

Comment:
Small nitpick but the wording here should say "vms" rather than "nodes" just to match the other doc strings.